environments/staging/terraform.yml

```diff
@@ -216,17 +216,17 @@ servers:
 
 
 proxy_servers:
-  - server_name: "proxy1-staging"
+  - server_name: "proxy2-staging"
```

Comment:
`proxy3-staging`?

Reply:
Sorry, nvm, I just noticed the "-" below -- it's a separate server, not a repeated key.